packages/tsconfig-reference/copy/en/options/tsBuildInfoFile.md

```diff
@@ -7,4 +7,17 @@ This setting lets you specify a file for storing incremental compilation informa
 building of larger TypeScript codebases. You can read more about composite projects [in the handbook](/docs/handbook/project-references.html).
 
 This option offers a way to configure the place where TypeScript keeps track of the files it stores on the disk to
-indicate a project's build state &mdash; by default, they are in the same folder as your emitted JavaScript.
+indicate a project's build state.
+
+The default depends on a combination of other settings
+
+ - If `outFile` the file is `<outFile>.tsbuildinfo`
+
```

Comment:
Does this format properly? Usually, I would expect to see this without newlines between the items.